docker-compose.yml

```diff
@@ -0,0 +1,5 @@
+api:
+  build: .
+  ports:
+    - '8080'
```

Comment:
I don't want to do this, because this will prevent docker-compose of launching multiples containers with this service.